opsdroid/loader.py

```diff
@@ -549,10 +549,10 @@ def _install_module(self, config):
 
         if self._is_module_installed(config):
             _LOGGER.debug(
-                _("Installed %s to %s"), config["name"], config["install_path"]
+                _("Installed %s to %s", "."), config["name"], config["install_path"], "."
```

Comment:
You've added too many dots. Remove the last one and remove the coma otherwise this log would look like this:

`Installed skill-hello to opsdroid,.`